### ISSUE TITLE
Collaborate with CKEditor 5 Angular 

### ISSUE THREAD

**Issue by naveedahmed1** [chymz/ng2-ckeditor#253]
With CKEditor 5 they are officially offering Angular support through https://github.com/ckeditor/ckeditor5-angular. 

I would suggest collaborating with them to make the official plugin more powerful and reliable.

They are also planning to offer support for Angular Material, may be you could contribute to that as well.

https://github.com/ckeditor/ckeditor5-angular/issues/21

**Comment by Rosyblackl** [comment]
thanks

**Comment by Rosyblackl** [comment]
Excuse me, Ckeditor print setup to cancel header page footer, you konw?

**Comment by kzimny** [comment]
@Rosyblackl: could you please explain a little bit more what you mean?
@naveedahmed1: npm [package ck-editor](https://www.npmjs.com/package/ckeditor) exists for years

**Comment by Rosyblackl** [comment]
When I print with ckeditor, I will print out the header and footer on the paper. I don't want headers and footers on paper.

**Comment by naveedahmed1** [comment]
@kzimny yes I know but I was referring to https://www.npmjs.com/package/@ckeditor/ckeditor5-angular official plugin from CEK team.
